celerdata/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: celerdata
+
+## Overview
+
+[CelerData][1] is the only SQL engine that is fast enough to run the most demanding workloads directly on your data lakehouse, all powered by StarRocks and cloud-managed by CelerData.
+
+StarRocks metrics enable database administrators to monitor their deployments. StarRocks provides a Prometheus-compatible information collection interface. 
+
+With this integration, we collect metrics and logs from StarRocks.
+
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the CelerData check on your host:
+
+1. Download and install the [Datadog Agent][8].
+2. To install the celerdata check on your host:
+
+   ```shell
+   datadog-agent integration install -t datadog-celerdata==<INTEGRATION_VERSION>
```

Comment:
This should include the integration version and replace <INTEGRATION_VERSION>

Reply:
done